internal/test/e2e/version_compatibility_test.go

```diff
@@ -1283,3 +1283,56 @@ func TestVersionCompatibility_Zipkin310OrNewer(t *testing.T) {
 		return err
 	})
 }
+
+func TestVersionCompatibility_KeysEntities(t *testing.T) {
+	cleanup := run.Koko(t)
+	defer cleanup()
+
+	dpCleanup := run.KongDP(kong.GetKongConfForShared())
+	defer dpCleanup()
+	require.NoError(t, util.WaitForKong(t))
+	require.NoError(t, util.WaitForKongAdminAPI(t))
+
+	admin := httpexpect.WithConfig(httpexpect.Config{
+		BaseURL:  "http://localhost:3000",
+		Reporter: httpexpect.NewRequireReporter(t),
+		Printers: []httpexpect.Printer{
+			httpexpect.NewCompactPrinter(t),
+		},
+	})
+
+	admin.POST("/v1/keys").WithJSON(&v1.Key{
+		Name: "second",
+		Jwk:  &v1.JwkKey{},
+	}).Expect().Status(http.StatusCreated)
+
+	t.Run("pre 3.0", func(t *testing.T) {
+		kongClient.RunWhenKong(t, "< 3.0.0")
+
+		util.WaitFunc(t, func() error {
+			return util.EnsureConfig(&v1.TestingConfig{})
```

Comment:
In order to make sure the configuration can be sync'd down, can you please add another entity (like, a `service`?) in the configuration and assert that it shows up in the DP?